examples/linearStaticCantileverAero/onsasExample_staticAeroLinearCantilever.m

```diff
@@ -0,0 +1,186 @@
+%md# Aerodynamic linear static cantilever beam example
+close all, clear all ;
+% add path
+addpath( genpath( [ pwd '/../../src'] ) );
+addpath( genpath( [ pwd ] ) );
+% material scalar parameters
+E = 70e9 ;  nu = 0.3 ; rho = 700 ; G = E / (2 * (1+nu))
+% geometrical scalar parameters
+l = 20 ; dext = .5 ;  b = 1e-3  ; dint  = dext - 2*b ;
+A = pi * (dext^2 - dint^2) / 4  ;
+J = pi * (dext^4 - dint^4) / 32 ; Iyy = J/2 ; Izz = Iyy ;
+Irho = diag([J Iyy Izz],3,3);
+% the number of elements of the mesh
+numElements = 10 ;
+%md##Numerical solution
+%md### MEBI parameters
+%md
+%md### materials
+%md Since the example contains only aeroFoone rod the fields of the `materials` struct will have only one entry. Although, it is considered constitutive behavior according to the SaintVenantKirchhoff law:
+materials.hyperElasModel  = '1DrotEngStrain' ;
+materials.hyperElasParams = [ E nu ] ;
+materials.density = rho ;
+%md
+%md### elements
+%md
+%mdTwo different types of elements are considered, node and beam. The nodes will be assigned in the first entry (index $1$) and the beam at the index $2$. The elemType field is then:
+elements(1).elemType = 'node'  ;
+elements(2).elemType = 'frame' ;
+%md for the geometries, the node has not geometry to assign (empty array), and the truss elements will be set as a rectangular-cross section with $t_y$ and $t_z$ cross-section dimensions in $y$ and $z$ directions, then the elemTypeGeometry field is:
+elements(2).elemTypeGeometry = [1 A J Iyy Izz Irho(1,1) Irho(2,2) Irho(3,3)] ;
+%md The drag and lift section function names are:
+elements(2).elemTypeAero   = [0 dext 0];
+elements(2).userDragCoef   = 'dragCoefFunction'   ;
+elements(2).userLiftCoef   = 'liftCoefFunction'   ;
+elements(2).userMomentCoef = 'momentCoefFunction' ;
+%md
+%md### boundaryConds
+%md
+%md The elements are submitted to two different BC settings. The first BC corresponds to a welded condition (all 6 dofs set to zero)
+boundaryConds(1).imposDispDofs = [ 1 2 3 4 5 6 ] ;
+boundaryConds(1).imposDispVals = [ 0 0 0 0 0 0 ] ;
+boundaryConds(2).loadsCoordSys = 'global'        ;
+boundaryConds(2).loadsTimeFact = @(t) 0;
+boundaryConds(2).loadsBaseVals = [ 0 0 0 -1 0 0 ] ;
+%md the name of the wind velocity function is: 
+boundaryConds(3).userWindVel    = 'windVel';
```

Comment:
Which is the most efficient way to call this function name from the aeroForce function?

Reply:
The question is in wich model struct should be saved that data and how to call it from assembler and aeroForce script

Reply:
i think we talked it could be on analysisSettings ... but we can discuss later

Reply:
Done